backup/Database.sql

```diff
@@ -44,7 +44,7 @@ UNLOCK TABLES;
 -- Table structure for table `Email`
 --
 
-DROP TABLE IF EXISTS `Email`;
+DROP TABLE IF EXISTS `emails`;
 /*!40101 SET @saved_cs_client     = @@character_set_client */;
 /*!50503 SET character_set_client = utf8mb4 */;
 CREATE TABLE `Email` (
```

Comment:
```suggestion
CREATE TABLE `emails` (
```
Shouldn't it also say emails here?